Composer/packages/client/src/components/Adapters/TeamsManifestGenerator.tsx

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { v4 as uuidv4 } from 'uuid';
+import { jsx } from '@emotion/core';
+import { DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
+import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+import formatMessage from 'format-message';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { IButtonStyles, IconButton } from 'office-ui-fabric-react/lib/components/Button';
+import { FontSizes, NeutralColors } from '@uifabric/fluent-theme/lib/fluent';
+import { useRef } from 'react';
+import { ITextField, TextField } from 'office-ui-fabric-react/lib/components/TextField';
+import { DialogTypes, DialogWrapper } from '@bfc/ui-shared/lib/components/DialogWrapper';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+
+import { defaultTeamsManifest } from '../../constants';
+
+const iconButtonStyle: IButtonStyles = {
+  root: {
+    height: 'unset',
+    float: 'right',
+    marginRight: '10px',
+  },
+  menuIcon: {
+    backgroundColor: NeutralColors.white,
+    color: NeutralColors.gray130,
+    fontSize: FontSizes.size16,
+  },
+  rootDisabled: {
+    backgroundColor: NeutralColors.white,
+  },
+  rootHovered: {
+    backgroundColor: 'unset',
+  },
+  rootPressed: {
+    backgroundColor: 'unset',
+  },
+};
+
+type TeamsManifestGeneratorProps = {
+  hidden: boolean;
+  botAppId?: string;
+  botDisplayName?: string;
+  onDismiss: () => void;
+};
+
+export const TeamsManifestGenerator = (props: TeamsManifestGeneratorProps) => {
```

Comment:
Add name dialog to the name of the component and the file since it's going to render a dialog

Reply:
Resolved